### ISSUE TITLE
[sigh] invalid option error for download_all

### ISSUE THREAD

**Issue by adomanico** [fastlane/fastlane#8231]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

Running the follow command results in an error on 2.15.1. The command succeeds on 2.14.2 after reverting.

```
fastlane sigh download_all -u my@email.com -i MYTEAMID -o ../build/
```

##### Complete output when running fastlane, including the stack trace and command used

```
invalid option -u
```

### Environment

```
### Stack

| Key                 | Value                                       |
| ------------------- | ------------------------------------------- |
| OS                  | 10.12.3                                     |
| Ruby                | 2.3.3                                       |
| Bundler?            | false                                       |
| Git                 | git version 2.10.1 (Apple Git-78)           |
| Installation Source | ~/.rbenv/versions/2.3.3/bin/fastlane        |
| Host                | Mac OS X 10.12.3 (16D32)                    |
| Ruby Lib Dir        | ~/.rbenv/versions/2.3.3/lib                 |
| OpenSSL Version     | OpenSSL 1.0.2j  26 Sep 2016                 |
| Is contained        | false                                       |
| Is homebrew         | false                                       |
| Xcode Path          | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version       | 8.2.1                                       |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |
```

**Comment by mfurtak** [comment]
Ah, sorry about this. 🙇 I think I understand the problem and will work on a fix!

**Comment by mfurtak** [comment]
The regression was introduced by https://github.com/fastlane/fastlane/pull/8129

**Comment by mfurtak** [comment]
A fix for this was released in [_fastlane_ 2.16.0](https://github.com/fastlane/fastlane/releases/tag/2.16.0) - thanks again for the report 🚀

**Comment by adomanico** [comment]
Awesome, great turnaround time 👍  thanks
